core/scripts/cli/sponsor/showMarketDetails.js

```diff
@@ -28,15 +29,30 @@ const showMarketDetails = async (web3, artifacts, emp) => {
       "Collateral:",
       fromWei(collateral.toString())
     );
+
+    const actions = {
+      back: "Back",
+      create: "Borrow more tokens",
+      redeem: "Repay tokens"
+    };
     const prompt = {
       type: "list",
       name: "choice",
       message: "What would you like to do?",
-      choices: [{ name: "Borrow more tokens" }, { name: backAction }]
+      choices: Object.values(actions)
     };
-    const input = await inquirer.prompt(prompt);
-    if (input["choice"] !== backAction) {
-      await create(web3, artifacts, emp);
+    const input = (await inquirer.prompt(prompt))["choice"];
+    switch (input) {
+      case actions.create:
+        await create(web3, artifacts, emp);
+        break;
+      case actions.redeem:
+        await redeem(web3, artifacts, emp);
+        break;
+      case actions.back:
```

Comment:
I like the `switch...case...case` structure much better for this, but I see why you didn't have it before when there were only 2 choices